### ISSUE TITLE
No accessible names - "2019 CHSS celebration of achievement.." + RSVP "here" + "See All Events" links

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#76]
This element has no HTML markup that would provide an accessible name for the element. A screen reader would either read nothing for this element or may make a guess.
